tests/lib/rules/use-v-on-exact.js

```diff
@@ -59,17 +74,195 @@ ruleTester.run('use-v-on-exact', rule, {
     },
     {
       code: `<template><button @click="foo"/><button @click.ctrl="foo"/></template>`
+    },
+    {
+      code: `<template><button v-on:click.exact="foo" v-on:click.ctrl="foo"/></template>`
+    },
+    {
+      code: `<template><a @mouseenter="showTooltip" @mouseenter.once="attachTooltip"/></template>`
+    },
+    {
+      code: `<template><input @keypress.exact="foo" @keypress.middle="bar" @keypress.ctrl="baz"></template>`
+    },
+    {
+      code: `<template><input @keypress.middle="foo" @keypress.middle="bar"></template>`
```

Comment:
wouldn't this throw a different kind of error? Both event listeners are the same

Reply:
ah, okay! I have never written any aslant plugin 😆